frontend/src/components/activePage/TableWorkspace/TableWorkspace.tsx

```diff
@@ -22,6 +28,8 @@ const TableWorkspace: FC<ITableWorkspaceProps> = ({ ...props }) => {
     expandAll,
     setCollapseAll,
     setExpandAll,
+    advanced,
```

Comment:
```suggestion
    showAdvanced,
```
Would make more obvious what this prop does